compiler/aarch64/codegen/ControlFlowEvaluator.cpp

```diff
@@ -534,3 +534,63 @@ OMR::ARM64::TreeEvaluator::ArrayCHKEvaluator(TR::Node *node, TR::CodeGenerator *
 	// TODO:ARM64: Enable TR::TreeEvaluator::ArrayCHKEvaluator in compiler/aarch64/codegen/TreeEvaluatorTable.hpp when Implemented.
 	return OMR::ARM64::TreeEvaluator::unImpOpEvaluator(node, cg);
 	}
+
+static TR::Register *
+commonMinMaxEvaluator(TR::Node *node, bool is64bit, bool isMax, TR::CodeGenerator *cg)
```

Comment:
Rather than an `isMax` boolean parameter, why not just pass in either `TR::CC_GT` or `TR::CC_LT` as a condition  code parameter depending on what the need is?

Reply:
Changed the code to pass `TR::ARM64ConditionCode` instead of `bool`.